blockchain/block.go

```diff
@@ -28,6 +27,9 @@ type BlockHeader struct {
 	Time uint32
 	// Nonce starts at 0 and increments by 1 for every hash when mining
 	Nonce uint64
+	// ExtraData is an extra field that can be filled with arbitrary data to
+	// be stored in the block
+	ExtraData []byte
```

Comment:
I was thinking its a standard blockchain feature, lets you store data for "eternity" and will potentially help us with future applications of our blockchain.